Makefile

```diff
@@ -400,10 +412,9 @@ $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TAR
 	| $(BUILD_OUTPUT_DIRECTORY)
 	./scripts/build/zip-file.sh -f $< -s $(TARGET_PLATFORM) -o $@
 
-$(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH).exe: \
-	$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH) \
-	| $(BUILD_OUTPUT_DIRECTORY) $(BUILD_TEMPORARY_DIRECTORY)
-	./scripts/build/electron-installer-nsis-win32.sh -n $(APPLICATION_NAME) -a $< -t $(BUILD_TEMPORARY_DIRECTORY) -o $@
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH).exe: \
+	| $(BUILD_DIRECTORY)
+	PATH=$(NODE_MODULES_BIN):$$PATH TARGET_ARCH=$(TARGET_ARCH) build --win nsis $(ELECTRON_BUILDER_OPTIONS)
```

Comment:
Hmmm, so no way to run electron-builder without adding things to the `$PATH`? :-/
Even so, it might be better to still explicitly call `$(NODE_MODULES_BIN)/build` rather than just `build`?